elixir/apps/api/lib/api/client/channel.ex

```diff
@@ -77,9 +77,14 @@ defmodule API.Client.Channel do
       :ok = Enum.each(actor_group_ids, &Policies.subscribe_to_events_for_actor_group/1)
       :ok = Policies.subscribe_to_events_for_actor(socket.assigns.subject.actor)
 
+      # Return all connected relays for the account
+      {:ok, relays} = select_relays(socket)
+      :ok = Enum.each(relays, &Domain.Relays.subscribe_to_relay_presence/1)
+
       :ok =
         push(socket, "init", %{
           resources: Views.Resource.render_many(resources),
+          relays: Views.Relay.render_many(relays, socket.assigns.subject.expires_at),
```

Comment:
Perhaps I didn't find it in the diff but is the same field present for the gateways too in their `init` message?

Reply:
Is the client now expected to filter this list based on RT latency? Or is the portal still sending proximity Relays based on geomapped IP?

Reply:
> Is the client now expected to filter this list based on RT latency? Or is the portal still sending proximity Relays based on geomapped IP?

Not as far as I know. We discussed that we can use all relays that are sent back because they will already be filtered by region / distance.

Reply:
@jamilbk we still load balance and only return 2 relays, the only thing changed is that we show it in a new place and broadcast updates.

@thomaseizinger yes, the structure is the same (and everywhere a new `id` field is added).

Reply:
Got it, thanks!